.hlint.yaml

```diff
@@ -39,18 +29,65 @@
 - warn: {lhs: Control.Exception.evaluate, rhs: evaluateWHNF}
 - warn: {lhs: Control.Exception.evaluate (force x), rhs: evaluateNF x}
 - warn: {lhs: Control.Exception.evaluate (x `deepseq` ()), rhs: evaluateNF_ x}
-
 - warn: {lhs: void (evaluateWHNF x), rhs: evaluateWHNF_ x}
 - warn: {lhs: void (evaluateNF x), rhs: evaluateNF_ x}
 
-- hint: {lhs: "foldl' (flip f)", rhs: "flipfoldl' f"}
+## Monad.Trans
+
+- warn: {lhs: flip runReaderT r t, rhs: usingReaderT r t}
```

Comment:
Hmm.. I think it's better to do _eta-reduction_ here. `hlint` can't handle `.` automatically. But it's okay to eta-reduce rules. So `flip runReaderT ===> usingReaderT` is better (because `flip runReaderT` can occur not fully applied in composition for example).